packages/amplify-category-function/src/provider-utils/awscloudformation/secrets/ssmClientWrapper.ts

```diff
@@ -108,17 +110,14 @@ export class SSMClientWrapper {
   };
 }
 
-const getSSMClient = async (context: $TSContext): Promise<aws.SSM> => {
-  try {
-    spinner.start();
-    spinner.text = 'Building and packaging resources';
-
-    const { client } = await context.amplify.invokePluginMethod(context, 'awscloudformation', undefined, 'getConfiguredSSMClient', [
-      context,
-    ]);
+const getSSMClient = async (context: $TSContext): Promise<SSM> => {
+  const { client } = await context.amplify.invokePluginMethod<{ client: SSM }>(
```

Comment:
The spinner here didn't describe anything useful so it was removed to reduce noise.